kernel/node/Node.h

```diff
@@ -34,6 +36,11 @@ struct FsNode : public RefCounted<FsNode>
 
     unsigned int master() { return _master; }
 
+    Mode get_mode() { return _mode; }
+
+    void set_mode(Mode mode) { _mode = mode; }
+
```

Comment:
There are too much spaces

Reply:
these spaces are also there for other functions

Reply:
@Keyboard-Slayer those spaces :rofl: